x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/advanced_view/detector_list.tsx

```diff
@@ -166,3 +172,7 @@ const DuplicateDetectorsWarning: FC<{ validation: Validation }> = ({ validation
     </Fragment>
   );
 };
+
+const DetectorIdentifier: FC<{ detector: Detector }> = ({ detector }) => {
```

Comment:
i'm not sure about the name `DetectorIdentifier`.
It's not obvious what it'll produce.
The component produces a string describing the detector. something like `StringifiedDetector`  is closer to what it's doing, but that's not great either.

Reply:
Renamed in f32ca31